Autopilot/AttitudeManager/SendInstructionsToSafety.cpp

```diff
@@ -0,0 +1,25 @@
+#include "SendInstructionsToSafety.hpp"
+#include "Interchip_A.h"
+#define PWM_CHANNELS 12
+
+int16_t pwmPercentages[PWM_CHANNELS] = {0};
+int16_t initialPWMPercentages[PWM_CHANNELS] = {0}; //TODO: put in initial PWM states in here. 
+
+
+void SendToSafety_Init(void)
+{
+    for(int i = 0; i < PWM_CHANNELS; i++)
+    {
+        pwmPercentages[i] = initialPWMPercentages[i];
+    }
+}
+
+SendToSafety_error_t SendToSafety_Execute(int channel, int percent)
+{
+    SendToSafety_error_t error;
+    error.errorCode = 0;
+    pwmPercentages[channel] = percent;
+
+    Interchip_SetPWM(pwmPercentages);
```

Comment:
I don't see a place where error.errorCode is changed if an error occurs.  Am I missing something?

Reply:
Yeah, this is a todo since interchip hasn't been written yet. I'll add it in a comment for now I guess. If interchip fails, it should return a status code, and that will feed into the error struct is the current plan iirc